lang/elixir/Makefile

```diff
@@ -0,0 +1,50 @@
+#
+# Copyright (C) 2020 Xiaobo Tian <peterwillcn@gmail.com>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=elixir
+PKG_VERSION:=1.10.3
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/elixir-lang/elixir/archive/v$(PKG_VERSION)
+PKG_HASH:=f3035fc5fdade35c3592a5fa7c8ee1aadb736f565c46b74b68ed7828b3ee1897
+
+PKG_LICENSE:=Apache-2.0
+PKG_LICENSE_FILES:=LICENSE
+PKG_MAINTAINER:=Xiaobo Tian <peterwillcn@gmail.com>
+
+PKG_INSTALL:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/elixir
+  SUBMENU:=Elixir
```

Comment:
Is this really needed? In that submenu, there is going to be just one package.

Reply:
Remove SUBMENU done